src/soho/dropdown/soho-dropdown.component.ts

```diff
@@ -387,7 +387,7 @@ export class SohoDropDownComponent implements AfterViewInit, AfterViewChecked, O
         // has been updated (assuming there is one), so
         // execute updated after angular has generated
         // the model and the view markup.
-        setTimeout(() => this.updated() );
+        setTimeout(() => this.updated());
```

Comment:
Timeout shouldn’t be required here. But it shouldn’t hurt either since it’s outside Ng zone.